packages/react/src/themes/teams/components/Status/statusStyles.ts

```diff
@@ -35,16 +35,33 @@ const getTextColor = (state: string, variables: StatusVariables) => {
   }
 }
 
-const statusStyles: ComponentSlotStylesInput<StatusPropsWithDefaults, StatusVariables> = {
+export const sizeToPxValue = {
+  smallest: 8,
+  smaller: 8,
+  small: 8,
+  medium: 10,
+  large: 12,
+  larger: 14,
+  largest: 16,
+}
+
+export const getSizeStyles = (sizeInPx: number, variables: StatusVariables) => {
+  const sizeInRem = pxToRem(sizeInPx + 2 * ((variables.borderColor && variables.borderWidth) || 0))
```

Comment:
```suggestion
  const borderWidth = (variables.borderColor && variables.borderWidth) || 0
  const sizeInRem = pxToRem(sizeInPx + 2 * borderWidth)
```

One line will not save us 😸 Can we make it more readable?